src/rpm/builder.rs

```diff
@@ -35,15 +36,20 @@ fn file_mode(_file: &fs::File) -> Result<u32, RPMError> {
     Ok(0)
 }
 
+static PAST_ERROR: &str = "1970 has already passed";
+static FUTURE_ERROR: &str =
+    "By 2100 we have a new, modern, package format spec and implementation. qed";
+
 fn date_time_as_u32(when: &chrono::DateTime<chrono::Utc>) -> u32 {
-    when.timestamp()
-        .try_into()
-        .expect("By 2100 we have a new, modern, package format spec and implementation. qed")
+    when.timestamp().try_into().expect(FUTURE_ERROR)
 }
 
-fn system_time_as_u32(when: std::time::SystemTime) -> u32 {
-    let dt = chrono::DateTime::<chrono::Utc>::from(when);
-    date_time_as_u32(&dt)
+fn system_time_as_u32(dt: std::time::SystemTime) -> u32 {
+    dt.duration_since(SystemTime::UNIX_EPOCH)
+        .expect(PAST_ERROR)
+        .as_secs()
+        .try_into()
+        .expect(FUTURE_ERROR)
```

Comment:
AFAIK we do not need `chrono` for converting `SytemTime` into UNIX timestamp, i.e. `std` should handle host time zone properly by itself.